Feat  user guide revision (#80)
ImperialCollegeLondon/icl-rcs-user-guide: docs/support/attend-a-clinic.md

# Attend a Clinic

> HPC code clinics are a good way to discuss your issue directly with an RCS team member. These sessions are normally held every Tuesday from 14:00 to 16:00 on Microsoft teams. If you're interested, [click here](https://outlook.office365.com/book/ResearchComputingService1@ImperialLondon.onmicrosoft.com/s/uFsEFdnBDkq3PlhXAmXO0Q2) to book a 40-minute time slot at least 48-hours in advance.

Reviewer: ```suggestion
HPC code clinics are a good way to discuss your issue directly with an RCS team member. These sessions are normally held every Tuesday from 14:00 to 16:00 on Microsoft teams. If you're interested, [click here](https://outlook.office365.com/book/ResearchComputingService1@ImperialLondon.onmicrosoft.com/s/uFsEFdnBDkq3PlhXAmXO0Q2) to book a 40-minute time slot. These are only available 48 hours in advance of each clinic.
```
